examples/openshift.json

```diff
@@ -0,0 +1,87 @@
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "OpenShift",
+      "openShiftConfig": {
+        "clusterUsername": "demo",
+        "clusterPassword": "demo"
+      }
+    },
+    "azProfile": {
+      "tenantId": "YOUR TENANT ID",
+      "subscriptionId": "YOUR SUBSCRIPTION ID",
+      "resourceGroup": "YOUR RESOURCE GROUP",
+      "location": "YOUR LOCATION"
+    },
+    "masterProfile": {
+      "count": 1,
+      "dnsPrefix": "YOUR DNS PREFIX",
+      "imageReference": {
+        "name": "YOUR IMAGE",
+        "resourceGroup": "YOUR IMAGE RESOURCE GROUP"
+      },
+      "storageProfile": "ManagedDisks",
+      "vmSize": "Standard_D4s_v3"
+    },
+    "agentPoolProfiles": [
+      {
+        "availabilityProfile": "AvailabilitySet",
+        "count": 1,
+        "imageReference": {
+          "name": "YOUR IMAGE",
+          "resourceGroup": "YOUR IMAGE RESOURCE GROUP"
+        },
+        "name": "compute",
+        "storageProfile": "ManagedDisks",
+        "vmSize": "Standard_D4s_v3"
+      },
+      {
+        "availabilityProfile": "AvailabilitySet",

```

Comment:
hmm can we have another with VMSS. We are trying to slowly walk away from av-set

Reply:
at the moment we're deploying openshift based on k8s 1.9 - therefore we were supposing that we couldn't use scalesets yet?

Reply:
Right. Should we add a validation for for using VMSS w/ OpenShift

Reply:
https://github.com/kargakis/acs-engine/pull/83 , thanks

Reply:
@jim-minter's changes pushed in this PR